server/functions/countShips.js

```diff
@@ -7,8 +7,50 @@ function countShips(cellId, arrayIn) {
     return !(i % 10) ? prev.concat([a.slice(i, i + 10)]) : prev
   }, []);
 
-  // console.log('array', arrayOut[0][9])
-
+  //-------------------------------------------------
+  let count = 0;
+//vertical
+  for (let i = 0; i < 10; i++) {
+    if (i !== +coordX) {
```

Comment:
I believe this cycle is supposed to count ships on vertical line. Why is it `if (i !== +coordX)` instead of `if (i === +coordX)`. It seems that it will count all ships expcept for the ships on the vertical line. Or am I missing something?

Reply:
This cycle counts all ships on vertical line except ship in coordX

Reply:
This cycle counts all ships on line except ship in coordX

Reply:
ok, got it